webapp/pages/post/_id/_slug/index.vue

```diff
@@ -110,10 +110,12 @@ export default {
     }
   },
   data() {
+    // Wolle: const { commentId = null } = this.$route.query
```

Comment:
```suggestion
```

Reply:
Ups.
I forgot to clean up. Sorry! 😚
Done in my work tree …